schema.ts

```diff
@@ -242,6 +242,12 @@ interface BaseSchema {
   /** @hidden */
   mutable?: boolean;
 
+  /**
```

Comment:
Can you also add a `// TODO(dweitzman): Use fixedId for the autocomplete name when available to support property renames.` above the `replaceInlineAutocompleteFunctionsWithNamedAutocompleteFunctions()` function definition in api.ts?

Reply:
Done